zkevm-circuits/src/state_circuit.rs

```diff
@@ -149,13 +160,30 @@ impl<F: Field> SubCircuitConfig<F> for StateCircuitConfig<F> {
         let lexicographic_ordering =
             LexicographicOrderingConfig::configure(meta, sort_keys, lookups, power_of_randomness);
 
+        let rw_permutation_config = PermutationChip::configure(
+            meta,
+            <RwTable as LookupTable<F>>::advice_columns(&rw_table),
+        );
+
         // annotate columns
         rw_table.annotate_columns(meta);
         mpt_table.annotate_columns(meta);
         u8_table.annotate_columns(meta);
         u10_table.annotate_columns(meta);
         u16_table.annotate_columns(meta);
 
+        <RwTable as LookupTable<F>>::columns(&rw_table)
+            .iter()
+            .for_each(|c| meta.enable_equality(*c));
```

Comment:
From the `enable_equality` I imagine that you're doing the fingerprint in another region and passing the cells via copy constraints.  I think the fingerprint could be implemented as a gadget applied inline at every row of the state circuit, to skip the copy constraints.  But this is an optimization, the current solution should work for now!

Reply:
You're right! Previously Rwtable continuity is carried over via a separate copy constraints, because the `PermutationChip` is meant to only cover fingerprints related logic. After your reminding, I think rwtable continuity can be realized via fingerprints as well, because first/last row of rwtable against fingerprint are one to one mapping, so we can leverage `PermutationChip` to do everything. 

I have done the change to simplify design in new commit https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1641/commits/de550345bb65f04250c65d6ac779484f29da4a54 so we can save copy contraints effort on rw_table itself.